sebastian-correa/tarea-1/index.html

```diff
@@ -6,58 +6,55 @@
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <div class="main-container">
-
-        <div class="top-info">
-            <span class="info">DOUBLE ISSUE</span>
-            <span class="info">OCT. 25 / NOV. 1, 2021</span>
-        </div>
-
-        <div class="magazine-cover">
-            <div class="title-container">
-                <span class="magazine-title">TIME</span>
-            </div>
-
-            <div class="img-container">
+    <main class="main-container">
+        <section class="top-info">
+            <p class="info">DOUBLE ISSUE</p>
+            <p class="info">OCT. 25 / NOV. 1, 2021</p>
+        </section>
+
+        <article class="magazine-cover">
+            <header class="title-container">
+                <h1 class="magazine-title">TIME</h1>
+            </header>
+
+            <figure class="img-container">
                 <img src="./assets/zuckerberg-testify.jpg" alt="" class="background-img">
-            </div>
+            </figure>
 
-            <div class="dialog-container">
+            <section class="dialog-container">
                 <div class="ios-dialog">
                     <div class="text-container">
-                        <span class="ios-text">Delete "Facebook"?</span>
+                        <p class="ios-text">Delete "Facebook"?</p>
                     </div>
                     
                     <div class="btn-container">
                         <div class="btn">
-                            <span class="cancel-btn ios-text">Cancel</span>
+                            <p class="cancel-btn ios-text">Cancel</p>
                         </div>
                         <div class="divider"></div>
```

Comment:
@Secorvil , this king of content empty elements should be Pseudo elements like ::after